### PR TITLE
WaveView Splash

### PR DESCRIPTION
**Pull request by htg2704**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,10 +1,12 @@
 apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 29
     buildToolsVersion "29.0.2"
     defaultConfig {
-        applicationId "com.android.splitpersonality"
+        applicationId "com.android.split_personality"
```
Comment (shreyshrey71) [inline app/build.gradle]: dont change this

**Review by shreyshrey71 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_create_profile.xml [hunk 2]
```diff
@@ -1,15 +1,87 @@
...
+
+    </RadioGroup>
+
+`   <Button
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:text="Save Profile"
+    android:layout_marginTop="700dp"
+    android:background="@drawable/gradient3"
+    android:textColor="#F8F6F6"
```
Comment (TejusR) [inline app/src/main/res/layout/activity_create_profile.xml:83]: Please do not hardcode. Save the value in Colors.xml and reference it here

File: app/src/main/res/layout/activity_create_profile.xml [hunk 3]
```diff
@@ -1,15 +1,87 @@
...
+            android:text="Mute"
+            android:textSize="18sp"/>
+
+    </RadioGroup>
+
+`   <Button
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:text="Save Profile"
+    android:layout_marginTop="700dp"
```
Comment (TejusR) [inline app/src/main/res/layout/activity_create_profile.xml:81]: Do not hardcode layout_marginTop. Use a LinearLayout or RelativeLayout inside the ScrollView

File: app/src/main/res/layout/activity_create_profile.xml [hunk 4]
```diff
@@ -1,15 +1,87 @@
...
-        android:text="Create Profile"/>
+        android:id="@+id/time_picker"
+        android:layout_marginTop="220dp"
+        />
+    <RadioGroup
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:id="@+id/radio_group"
+        android:gravity="center"
+        android:layout_marginTop="600dp">
```
Comment (TejusR) [inline app/src/main/res/layout/activity_create_profile.xml:56]: Do not hardcode layout_marginTop. Use a LinearLayout or RelativeLayout inside the ScrollView.

File: app/src/main/res/layout/activity_create_profile.xml [hunk 5]
```diff
@@ -1,15 +1,87 @@
...
+        android:textSize="25sp"
+        android:gravity="center"
+        android:textColor="@color/black"
+        android:text="Alarm"/>
+    <TimePicker
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Create Profile"/>
+        android:id="@+id/time_picker"
+        android:layout_marginTop="220dp"
```
Comment (TejusR) [inline app/src/main/res/layout/activity_create_profile.xml:49]: Do not hardcode layout_marginTop. Use a LinearLayout or RelativeLayout inside the ScrollView.

**Review by TejusR — COMMENTED** [review]
Do not hardcode colors. Move the color values to Colors.xml and reference it.

**Pull request closed without merging**